ballista/core/src/utils.rs

```diff
@@ -36,6 +37,64 @@ use std::{fs::File, pin::Pin};
 use tonic::codegen::StdError;
 use tonic::transport::{Channel, Error, Server};
 
+#[derive(Debug, Clone)]
+pub struct GrpcClientConfig {
+    pub connect_timeout_seconds: u64,
+    pub timeout_seconds: u64,
+    pub tcp_keepalive_seconds: u64,
+    pub http2_keepalive_interval_seconds: u64,
+    pub keepalive_timeout_seconds: u64,
+}
+
+impl GrpcClientConfig {
+    pub fn from_ballista_config(config: &BallistaConfig) -> Self {
```

Comment:
would it make sense to implement this as `From` or `Into` not sure which one would be better